src/nvim/spell.c

```diff
@@ -2487,7 +2487,9 @@ buf_T *open_spellbuf(void)
 
   buf->b_spell = true;
   buf->b_p_swf = true;        // may create a swap file
-  ml_open(buf);
+  if (ml_open(buf) == FAIL) {
+    ELOG("Error opening a new memline");
```

Comment:
`ml_open` already does `IEMSG` if it fails, so we don't need to log or do anything, only return.

But I think the nature of open_spellbuf is that it never fails, which is why the error wasn't checked here. If we return early then we would return nonsense.

So this should be changed to ignore the return value, i.e.

    (void)ml_open(buf);